### PR TITLE
Changing picture hyperlinks to download warp

### PR DESCRIPTION
**Pull request by elviskahoro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
   <a href="https://warp.dev/blog">Blog</a>
 </p>
 
-<a href="https://www.youtube.com/watch?v=T7R8lvvBgOI">
+<a href="https://app.warp.dev/get_warp">
```
Comment (dannyneira) [inline README.md:15]: i think we should still keep the demo link as before

**Review by dannyneira — COMMENTED** [review]
I think we should leave lines 2 and 15 as they were and just change the install/download button as you did

File: README.md (hunk 1 above)
Comment (elviskahoro) [reply README.md:15]: got it

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <p align="center">
-    <a href="https://warp.dev">
+    <a href="https://app.warp.dev/get_warp">
```
[the remark this answers is not in the document]
Comment (elviskahoro) [reply README.md:2]: Let's keep the youtube link but change the Warp button.
CleanShot 2022-08-29 at 13 51 59@2x

[the remark this answers is not in the document]
Comment (dannyneira) [reply README.md:2]: that works, as long as we have the homepage somewhere

**Comment by dannyneira** [comment]
good to merge

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (elviskahoro) [reply README.md:2]: yeah the homepage is under about

**Pull request merged**